index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav">
+        <a href="">video</a>
+        <a href="">sidebar</a>
+    </nav>
+    
+    
+    
+    <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/Yg6UhhV_K1s" 
+            title="YouTube video player" 
+            frameborder="0" 
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+            allowfullscreen controls>
+    </iframe>
```

Comment:
Kindly use the required semantic tag for this section as this helps better define the different sections of your page and improves your page's section engine optimization

Reply:
- [ ] **Hi, you're doing great with the review but I want to get to know why you said you should use the semantic tag as the `<nav>` tag is a semantic tag. Could you please explain more in detail, which semantic tag you should use as it is very confusing to understand your review in this context which is contrary to the highlighted line of code?**